### DISCUSSION TITLE
Maximum number of devices

### DISCUSSION THREAD

**Discussion by Dani-Hg** [vintlabs/fauxmoESP#150] [Q&A]
I would like to know the maximum number of devices that can be used. I noticed that with a NODEMCU (ESP8266) a maximum of 10 devices are possible and with an ESP32-WROOM-32 there are 20 devices. But I need 30 or more. Is there a possibility to change that?

I'm using Arduino 1.8.13
FauxmoESP 3.2.0
and currently downloaded AsyncTCP-master from "me-no-dev" 
and of course I have "IwIP 1.4 Higher Bandwidth" in the settings. 
Use the fauxmoESP-Basic example where I only added more devices.

**Comment by pvint** [comment]
I was tempted to simply call this a limitation, but it's bugging me too! I've looking into it. (See issue #152 )

**Comment by hennenzac** [comment]
I'm a little confused on the max number of devices?  I see this person mentions 10, but in other sources I read it's a maximum of 3, as mentioned in issue #149.  I have been having trouble getting the devices discovered at all.  Almost seems random when it works and when it doesn't.  I can't seem to get more than 2 devices.  If I have 3 devices with the .addDevice, nothing shows up (so far...).

**Reply by Sofakanten** [reply]
I was able to connect up to 9 devices successfully. However, whenever I attempted to add more, Alexa seemed unable to discover them. Interestingly, reverting back to the initial setup of nine devices, everything appeared to operate smoothly again. The issue seems to occur somewhat randomly.
